client/src/components/SponsorPage/SponsorsPage.jsx

```diff
@@ -129,11 +132,11 @@ class SponsorsPage extends React.Component {
       <Page title="Sponsors" {...this.props}>
         <FirstSectionContainer>
           <Container>
```

Comment:
just a `Container` is a very general name, can we try a naming with more meaning of what it does? like if it center stuff maybe, `AlignCenter` or `AlignHorizontal` or `AlignVertical` etc